### PR TITLE
add details information for Arm64 in README

### PR DESCRIPTION
**Pull request by lubinsz**
Signed-off-by: Bin Lu <bin.lu@arm.com>

### CODE REVIEW

**Comment by marquiz** [comment]
Just a small nitpick. I'd alter the commit message a bit (at least start with a capital letter), e.g. something like:
"Add Arm64 details into the README"

Otherwise looks good to me

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ such as restricting discovered features with the --label-whitelist option._
 | RDTL3CA        | Intel L3 Cache Allocation Technology
 | RDTL2CA        | Intel L2 Cache Allocation Technology
 
-### CPUID Features (Partial List)
+### Intel CPUID Features (Partial List)
```
Comment (balajismaniam) [inline README.md]: s/Intel/x86

File: README.md [hunk 2]
```diff
@@ -112,6 +112,20 @@ such as restricting discovered features with the --label-whitelist option._
 | SSE4.2         | Streaming SIMD Extensions 4.2 (SSE4.2)
 | SGX            | Software Guard Extensions (SGX)
 
+### Arm64 CPUID Features (Partial List)
+
+| Feature name   | Description                                                  |
+| :------------: | :----------------------------------------------------------: |
+| AES            | Announcing the Advanced Encryption Standard
```
Comment (balajismaniam) [inline README.md]: Is this AES different from x86 AES? If not please remove it from this list. Same comment holds for rest of the feature list.

Comment (lubinsz) [reply README.md]: Hi @balajismaniam 
For same features, There are different names on the two platforms.
In x86 platform, it is defined as following:
AESNI:       "AESNI",       // Advanced Encryption Standard New Instructions
In Arm platform:
AES            | Announcing the Advanced Encryption Standard
For more reference, please see the file as reference:
https://github.com/lubinsz/cpuid/blob/master/cpuid.go

File: README.md (hunk 1 above)
Comment (lubinsz) [reply README.md]: @balajismaniam  OK

File: README.md (hunk 2 above)
Comment (balajismaniam) [reply README.md]: Got it. Thanks.

**Comment by balajismaniam** [comment]
LGTM.
@lubinsz Please update this branch.

**Comment by balajismaniam** [comment]
@lubinsz needs rebase.

**Pull request closed without merging**